src/components/Banner/index.tsx

```diff
@@ -1,34 +1,28 @@
 import React from 'react';
 import './Banner.scss';
+import config from '../../config.json';
 
-const bannerLink =
-  'https://nodejs.org/en/blog/vulnerability/february-2020-security-releases/';
+const bannerBtnText = 'Blog post';
```

Comment:
Does this ever change? @benhalverson . If so i can put it in the `config.json`

Reply:
I think yes

Reply:
ok ill add it in `config.json` then. Thanks!

Reply:
Yes I think so

Reply:
Added in